db/migrate/20220115053923_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.references :user, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.string :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] The `likes_counter` has to be t.integer instead of t.string
- [ ] You are missing the ` t.integer :comments_counter`